providers/os/detector/detector_win.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package detector
+
+import (
+	"strconv"
+
+	"github.com/rs/zerolog/log"
+	"go.mondoo.com/cnquery/v11/providers-sdk/v1/inventory"
+	"go.mondoo.com/cnquery/v11/providers/os/connection/shared"
+	win "go.mondoo.com/cnquery/v11/providers/os/detector/windows"
+	"go.mondoo.com/cnquery/v11/providers/os/registry"
+)
+
+func runtimeWindowsDetector(pf *inventory.Platform, conn shared.Connection) (bool, error) {
+	data, err := win.GetWmiInformation(conn)
+	if err != nil {
+		log.Debug().Err(err).Msg("could not gather wmi information")
```

Comment:
why are we not returning the error here? 

Is it safe for us to access `data` even if there is an error?

Reply:
whoops, i left that one out, great catch